lib/bulbs-elements/util/in-view-monitor.js

```diff
@@ -128,7 +138,9 @@ function handleDisplayMutation () {
       });
 
       Object.keys(monitoredItems).forEach((key) => {
-        maybeEmitEvent(monitoredItems[key]);
+        if(monitoredItems[key]) {
```

Comment:
Is this `if` conditional actually needed?  If we're already iterating over the keys of the object, they should always be there, no?